[rospy.client][INFO] 2020-11-11 03:57:28,714: init_node, name[/train_6946_1605067048707], pid[6946]
[xmlrpc][INFO] 2020-11-11 03:57:28,715: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2020-11-11 03:57:28,716: Started XML-RPC server [http://luigi:39215/]
[rospy.init][INFO] 2020-11-11 03:57:28,716: ROS Slave URI: [http://luigi:39215/]
[rospy.impl.masterslave][INFO] 2020-11-11 03:57:28,716: _ready: http://luigi:39215/
[rospy.registration][INFO] 2020-11-11 03:57:28,718: Registering with master node http://192.168.0.201:11311
[xmlrpc][INFO] 2020-11-11 03:57:28,718: xml rpc node: starting XML-RPC server
[rospy.init][INFO] 2020-11-11 03:57:28,817: registered with master
[rospy.rosout][INFO] 2020-11-11 03:57:28,817: initializing /rosout core topic
[rospy.rosout][INFO] 2020-11-11 03:57:28,823: connected to core topic /rosout
[rospy.simtime][INFO] 2020-11-11 03:57:28,824: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2020-11-11 03:57:28,836: topic[/bev/flow_image] adding connection to [http://192.168.0.201:38315/], count 0
[rospy.internal][INFO] 2020-11-11 03:57:28,850: topic[/bev/occupancy_image] adding connection to [http://192.168.0.201:38315/], count 0
[rospy.internal][INFO] 2020-11-11 03:57:28,854: topic[/odom] adding connection to [http://192.168.0.201:46537/], count 0
[rospy.internal][INFO] 2020-11-11 03:57:29,088: topic[/rosout] adding connection to [/rosout], count 0
[rospy.core][INFO] 2020-11-11 03:57:31,295: signal_shutdown [atexit]
[rospy.internal][INFO] 2020-11-11 03:57:31,299: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2020-11-11 03:57:31,300: topic[/bev/flow_image] removing connection to http://192.168.0.201:38315/
[rospy.internal][INFO] 2020-11-11 03:57:31,300: topic[/bev/occupancy_image] removing connection to http://192.168.0.201:38315/
[rospy.internal][INFO] 2020-11-11 03:57:31,301: topic[/odom] removing connection to http://192.168.0.201:46537/
[rospy.impl.masterslave][INFO] 2020-11-11 03:57:31,301: atexit
[rospy.internal][WARNING] 2020-11-11 03:57:31,302: Unknown error initiating TCP/IP socket to 192.168.0.201:41223 (http://192.168.0.201:38315/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

[rospy.internal][WARNING] 2020-11-11 03:57:31,303: Unknown error initiating TCP/IP socket to 192.168.0.201:54639 (http://192.168.0.201:46537/): Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 562, in connect
    self.read_header()
  File "/opt/ros/noetic/lib/python3/dist-packages/rospy/impl/tcpros_base.py", line 657, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
AttributeError: 'NoneType' object has no attribute 'buff_size'

